### DISCUSSION TITLE
Connect not working in many to many relationship

### DISCUSSION THREAD

**Discussion by tomwray13** [prisma/prisma#11975] [Q&A]
Hey there!

I have a relationship in the project I'm working on where a dashboard can have multiple invites and an invite can have multiple dashboards.

So I've setup an [implicit many to many relationship](https://www.prisma.io/docs/concepts/components/prisma-schema/relations/many-to-many-relations#implicit-many-to-many-relations) in my prisma schema that looks like this:

```
model Dashboard {
  id        Int          @id @default(autoincrement())
  invites   UserInvite[]
  ... other fields
}

model UserInvite {
  id         String      @id @default(cuid())
  dashboards Dashboard[]
  ... other fields
}
```

When I try to write a prisma function to create a new UserInvite and connect multiple existing dashboards, my IDE doesn't like it:

<img width="443" alt="image" src="https://user-images.githubusercontent.com/24568306/155102504-6a5d43e5-3b54-4ecb-a889-6d5ea7591221.png">

Here is part of the error message shown:

<img width="828" alt="image" src="https://user-images.githubusercontent.com/24568306/155102597-7dbb78ca-3cb7-4062-b12e-e4036e1ee020.png">

Here is what I can see in the [connect docs](https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#connect) (which is what I've implemented above):

<img width="707" alt="image" src="https://user-images.githubusercontent.com/24568306/155103876-858f70d0-9933-42cc-80be-fca11f069c6f.png">

Any help would be really appreciated!

**Comment by arthurlomakin11** [comment]
Can you print screen error at the end of it?

**Reply by tomwray13** [reply]
here you go!

<img width="788" alt="image" src="https://user-images.githubusercontent.com/24568306/155124151-a49a3896-f23e-4d2e-8af0-923ba0f5770f.png">

**Reply by arthurlomakin12345** [reply]
Hmm... You can use .create({data: ...} **as any**) if you want. But you will lose typing here.

**Comment by arthurlomakin11** [comment]
Scroll down to the error in VS Code

**Comment by nurul3101** [comment]
Hey @tomwray13 👋🏻 ,
Can you please share your schema file with all the fields of `Dashboard` and `UserInvite` model? 
I can try to reproduce it on my end and see what's causing the issue.

**Reply by vladi-stevanovic** [reply]
👋 Hello @tomwray13  I'm going through pending GitHub Discussions to check if you need any help. Is this issue still open?
Please let us know if we can investigate it further or if you've already solved it, we'd love to know how!

It would be super useful for other users facing this problem! 🙌

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
